src/data_collector/polygon_data/data_storage.py

```
@@ -519,97 +507,30 @@ def store_tickers(
 
-                for row in result:
-                    ticker_dict = dict(row._mapping)
-                    tickers.append(ticker_dict)
+            for row in result:
+                ticker_dict = dict(row)
+                tickers.append(ticker_dict)
 
-                logger.info(f"Retrieved {len(tickers)} tickers from database")
-                return tickers
+            logger.info(f"Retrieved {len(tickers)} tickers from database")
+            return tickers
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)